components/org.wso2.carbon.business.rules.web/src/components/common/error/ErrorDisplay.jsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import AccessDenied from './AccessDenied';
+import Forbidden from './Forbidden';
+import InternalServerError from './InternalServerError';
+import UnknownError from './UnknownError';
+
+/**
+ * Represents an Error Message Display
+ */
+export default class ErrorDisplay extends Component {
+    /**
+     * Returns the respective Error Component, with the given error code
+     * @param {number} errorCode        Error Code
+     * @returns {Component}             Error Component
+     */
+    displayError(errorCode) {
+        switch (errorCode) {
+            case 401:
+                return <AccessDenied />;
+            case 403:
+                return <Forbidden />;
+            case 500:
+                return <InternalServerError />;
+            default:
+                return <UnknownError />;
+        }
+    }
+
+    render() {
+        return (
+            <center>
```

Comment:
Prefer CSS styles over `<center>`

Reply:
I will send this kind of CSS fixes in a separate PR